static/scripts/login.js

```diff
@@ -47,9 +47,16 @@ $(document).ready(() => {
 	const $school = $('.school');
 	const $systems = $('.system');
 	const $modals = $('.modal');
+	const $oauthButton = $('.oauth-button-group');
 	const $pwRecoveryModal = $('.pwrecovery-modal');
 	const $submitButton = $('#submit-login');
 
+	let selectedSystem = null;
+
+	// initial hide of the systems select
+	$systems.parent().hide();
```

Comment:
Can the form fields be hidden by having css classes added in html directly?

Reply:
yes, it might be possible, it didnt work as i have tried it, but i will try again